docs/content/routing/services/index.md

```diff
@@ -9,6 +9,34 @@ The `Services` are responsible for configuring how to reach the actual services
 
 ## Configuration Example
 
+??? example "Specify the port that traefik will use to connect to the service"
+
+    ```yaml tab="Docker"
+    # Sets the port traefik will use to connect to the container
+    labels:
+    - "traefik.http.services.my-service.loadbalancer.server.port=12345"
+    ```
```

Comment:
```suggestion
```

Reply:
Could you remove the Docker example from here and move it to the "Docker routing" section please (this structural change was merged after you opened your PR :) ).